### PR TITLE
Add users database page

### PR DESCRIPTION
**Pull request by kevincela**
- Add users database page

### CODE REVIEW

File: src/StandaloneApp.vue [hunk 1]
```diff
@@ -123,7 +123,8 @@ function configureAxios() {
         }
       } else {
         // show error notification
-        notificationsStore.createNotificationFromAxiosError(error)
+        if (!(error instanceof CanceledError))
+          notificationsStore.createNotificationFromAxiosError(error)
```
Comment (Tbaile) [inline src/StandaloneApp.vue:127]: Does prettier allows this? When possible, add brackets please :pray:

File: src/components/standalone/users_database/CreateOrEditUserDrawer.vue [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  NeSideDrawer,
+  NeInlineNotification,
+  getAxiosErrorMessage
+} from '@nethserver/vue-tailwind-lib'
+import { ValidationError, ubusCall } from '@/lib/standalone/ubus'
+import type { User } from './UsersDatabaseManager.vue'
+
+const props = defineProps<{
+  isShown: boolean
+  itemToEdit: User | null
```
Comment (Tbaile) [inline src/components/standalone/users_database/CreateOrEditUserDrawer.vue:22]: You can use this
```suggestion
  itemToEdit?: User
```
and even forget to set it every time

File: src/StandaloneApp.vue (hunk 1 above)
Comment (Tbaile) [inline src/StandaloneApp.vue:127]: Does prettier allows this? When possible, add brackets please :pray:

File: src/components/standalone/users_database/CreateOrEditDatabaseDrawer.vue [hunk 3]
```diff
@@ -0,0 +1,447 @@
...
+  user_attr: string
+  user_cn: string
+  start_tls: string | boolean
+  tls_reqcert: string
+  description: string
+}
+
+const props = defineProps<{
+  isShown: boolean
+  itemToEdit: UserDatabase | null
```
Comment (Tbaile) [inline src/components/standalone/users_database/CreateOrEditDatabaseDrawer.vue:41]: As above

**Review by Tbaile — CHANGES_REQUESTED** [review]

File: src/StandaloneApp.vue (hunk 1 above)
Comment (Tbaile) [reply src/StandaloneApp.vue:127]: Indeed it's been allowed by eslint, not good, you can leave it as it is, gonna open a PR afterwards

Comment (kevincela) [reply src/StandaloneApp.vue]: whoops I've already added the brackets, don't think it's going to cause an issue though :P

**Review by Tbaile — APPROVED** [review]

**Pull request merged**